.github/workflows/coverage.yml

```
@@ -0,0 +1,23 @@
+name: Code coverage
+
+on:
```

`push` などを用いて、ベースブランチのcoverageを測ることでPRでのcoverage変動を見ることができるようになります。

```suggestion
on:
  pull_request:
    branches: ["main"]
  push:
    branches: ["main"]
```